crypto/modes/asm/ghash-s390x.pl

```diff
@@ -251,6 +255,7 @@
 .type	rem_4bit,\@object
 .size	rem_4bit,(.-rem_4bit)
 .string	"GHASH for s390x, CRYPTOGAMS by <appro\@openssl.org>"
+.comm	OPENSSL_s390xcap_P,S390X_CAPLEN,8
```

Comment:
i added the definition because all other asm modules have it. Is there a reason to not completely remove this from the asm modules and have the definition in s390xcap.c ?

Reply:
Original rationale was to minimize barrier when taking modules to other projects. I mean with .comm in place you'll always be able to link. However! At *that* point modules were calling query instructions upon each call. *Now* when OPENSSL_s390xcap_P caches queries and is version specific, having per-module .comm actually has potential to cause more harm than do good. So don't add it to ghash-s390x.pl and *remove* it from other modules. Thanks for bringing this up!

Reply:
Let's even put it this way. It's actually appropriate to remove .comm directives even from 1.1.0. So could you make it either as separate commit that will be cherry-picked to 1.1.0 or make it separate request when we're done with this one?

Reply:
PR #4563